eval_results/ablation/no_agent_2/03_Ambient_Lead_multi.html

```
@@ -1,172 +1,166 @@
+```json
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)